wazo_call_logd/plugins/retention/plugin.py

```diff
@@ -4,16 +4,26 @@
 from .http import RetentionResource
 from .services import RetentionService
 from .notifier import RetentionNotifier
+from .listener import RententionListener
+
+from wazo_call_logd.bus import BusConsumer, BusPublisher
+from wazo_call_logd.database.queries import DAO
+from flask_restful import Api
```

Comment:
Same comment for import sorting as above (and for the copyright - the date is 2021 but it should be updated to 2021-2023)